Assets/SEE/Controls/Actions/ResizeNodeAction.cs

```diff
@@ -430,17 +459,98 @@ private void InitHandles()
+                    Texture texture = AssetDatabase.LoadAssetAtPath<Texture2D>(resizeArrowBottomRightTexture);
+                    material = Materials.New(Materials.ShaderType.Sprite, Color.white, texture, (int)RenderQueue.Overlay);
+                }
+                else
+                {
+                    handle.transform.localRotation = Quaternion.Euler(0f, direction.x > 0f ? 180f : 0f + direction.z * 90f, 0f);
+                    Texture texture = AssetDatabase.LoadAssetAtPath<Texture2D>(resizeArrowRightTexture);
+                    material = Materials.New(Materials.ShaderType.Sprite, Color.white, texture, (int)RenderQueue.Overlay);
+                }
+                handle.GetComponent<Renderer>().material = material;
+                handle.transform.localPosition = new(
+                        parentWorldPosition.x + 0.5f * parentWorldScale.x * direction.x,
+                        parentWorldPosition.y
+                                + (direction == Vector3.up ? 0.5f * parentWorldScale.y * direction.y + 0.5001f * handle.WorldSpaceSize().y : 0f),
+                        parentWorldPosition.z + 0.5f * parentWorldScale.z * direction.z);
+
                 handle.name = $"handle__{direction.x}_{direction.y}_{direction.z}";
                 handle.transform.SetParent(transform);
+                Portal.InheritPortal(from: transform.gameObject, to: handle);
                 return handle;
             }
 
+            /// <summary>
+            /// Update the up/down handle's position and rotation so that it faces the camera.
+            /// </summary>
+            private void UpdateUpDownHandlePosition()
+            {
+                if (mainCameraTransform == null || upDownHandleTransform == null)
+                {
+                    return;
+                }
+
+                Vector3 directionToCamera = mainCameraTransform.position - transform.position;
+                if (Mathf.Abs(directionToCamera.x) > Mathf.Abs(directionToCamera.z))
+                {
+                    directionToCamera = Vector3.Normalize(new(directionToCamera.x, 0f, 0f));
+                    if (upDownHandleTransform.localRotation.eulerAngles.y == 0f)
+                    {
+                        upDownHandleTransform.localRotation = Quaternion.Euler(90f, 90f, 0f);
+                        flipScales();
+                    }
+                }
+                else
+                {
+                    directionToCamera = Vector3.Normalize(new(0f, 0f, directionToCamera.z));
+                    if (upDownHandleTransform.localRotation.eulerAngles.y != 0f)
+                    {
+                        upDownHandleTransform.localRotation = Quaternion.Euler(90f, 0f, 0f);
+                        flipScales();
+                    }
+                }
+
+                // The height is updated during the resize process
+                upDownHandleTransform.localPosition = new(
+                        0.5001f * directionToCamera.x,
```

Comment:
This `0.5001f` magic number is repeated a few times in this file, can you extract it into a named constant?